Lab1/app/build.gradle

```diff
@@ -0,0 +1,59 @@
+apply plugin: 'com.android.application'
+apply from: 'git.gradle'
+def versionArray = getArray()
+
+android {
+    signingConfigs {
+        relase {
+            storeFile file('C:\\Users\\Andrew\\Documents\\Android store\\Keys.jks')
+            storePassword 'MiaSanRomen'
+            keyAlias 'Converter'
+            keyPassword 'MiaSanRomen'
+        }
+    }
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
+
+    defaultConfig {
+        applicationId "com.example.myapplication"
+        minSdkVersion 14
+        targetSdkVersion 30
+        versionCode generateVersionCode(versionArray)
+        versionName "${versionArray[0]}.${versionArray[1]}.${versionArray[2]}"
+        versionName "1.0"
```

Comment:
?

Reply:
Please check PPO-Labs/Lab1/app/src/main/java/com/example/myapplication/
Probably you were in folders of my "demo version" or "full version".